### PR TITLE
add v1.6 article

### PR DESCRIPTION
**Pull request by narimiran**
This one supersedes #301.

- `Other features` is now link to the full changelog in Nim repo (https://github.com/nim-lang/Nim/pull/18932)
- several minor improvement throughout the article

### CODE REVIEW

File: jekyll/_posts/2021-09-30-version-160-released.md [hunk 1]
```diff
@@ -0,0 +1,417 @@
...
+
+
+# Footnotes
+
+Tested on a 2.3 GHz 8-Core Intel Core i9, 2019 macOS 11.5 with 64GB RAM.
+* [1] command used: `nim c -d:danger`.
+  The binary size can be further reduced to 49K with stripping `--passL:-s`
+  and link-time optimization (`--passC:-flto`).
+  Statically linking against `musl` brings it down to 5K, see
+  [link](https://irclogs.nim-lang.org/07-07-2020.html#12:31:34).
```
Comment (Yardanico) [inline jekyll/_posts/2021-09-30-version-160-released.md:408]: As I've said in the original PR, I'm not sure if we should link that exact discussion - it uses some tricks like `--os:any -d:posix` or `-d:noSignalHandler`

Comment (Araq) [reply jekyll/_posts/2021-09-30-version-160-released.md:408]: These are not "tricks", it's things you can easily find out if you need to really care and if it's not about a silly pissing contest. I would phrase it differently though or remove it altogether. It matters for embedded development, so measure it for an embedded target...

Comment (dom96) [reply jekyll/_posts/2021-09-30-version-160-released.md:408]: Why not link to this instead? https://hookrace.net/blog/nim-binary-size/

Comment (Yardanico) [reply jekyll/_posts/2021-09-30-version-160-released.md:408]: @dom96 this is pretty outdated (we have os:any now for example) and really specific to IMO

**Comment by narimiran** [comment]
Superseded by #304. (And the original PR by @timotheecour is now merged)

**Pull request closed without merging**